homeassistant/components/modbus/strings.json

```diff
@@ -70,6 +70,10 @@
     }
   },
   "issues": {
+    "deprecated_lazy_error_count": {
+      "title": "`{config_key}` configuration key is being removed",
+      "description": "Please remove the `{config_key}` key from the {integration} entry in your configuration.yaml file and restart Home Assistant to fix this issue."
```

Comment:
May be explain what is no longer working now?

Reply:
Done... I hope the "git push" worked.

Reply:
Some small spelling and quote things.